local/compose.go

```diff
@@ -558,6 +558,10 @@ func (s *composeService) ensureNetwork(ctx context.Context, n types.NetworkConfi
 	_, err := s.apiClient.NetworkInspect(ctx, n.Name, moby.NetworkInspectOptions{})
 	if err != nil {
 		if errdefs.IsNotFound(err) {
+			if n.External.External {
+				return fmt.Errorf("Network %s declared as external, but could not be found", n.Name)
```

Comment:
Shouldn't the linter fail on an error that starts with uppercase?

Reply:
fixed. Indeed I thought linter did detect these things, it seems not (just double checked the linter is active in /local).